azure-iot-common/tests/test_asyncio_compat.py

```diff
@@ -0,0 +1,79 @@
+import pytest
```

Comment:
>pytest [](start = 7, length = 6)

what is the "event_loop" parameter in your calls.  It looks like a fixture, but I don't see if defined anywhere in here and I don't see how it could be imported.  What crazy magic is this?

Reply:
pytest-asyncio

---
In reply to: [251159459](https://github.com/Azure/azure-iot-sdk-python-preview/pull/43#discussion_r251159459) [](ancestors = 251159459)